open-broker-common/src/main/java/de/fraunhofer/iais/eis/ids/broker/core/common/persistence/ResourcePersistenceAndIndexing.java

```diff
@@ -164,6 +168,7 @@ public URI updated(Resource resource, URI connectorUri) throws IOException, Reje
         }
         if (!repositoryFacade.graphIsActive(connectorUri.toString())) {
             connectorUri = URI.create(componentCatalogUri.toString() + connectorUri.hashCode());
+            logger.info("Connector Uri",connectorUri);
```

Comment:
This logging information is not descriptive enough to be of any use. Can you please either add a more descriptive text and more information or otherwise remove it?